src/main/groovy/au/org/ala/ws/security/authenticator/AlaOidcAuthenticator.groovy

```diff
@@ -114,10 +114,14 @@ class AlaOidcAuthenticator extends UserInfoOidcAuthenticator {
                 new JWTClaimsSet.Builder().issuer(issuer.value).build(),
                 requiredClaims?.toSet())
 
+        Collection<String> accessTokenRoles
+
         try {
 
             JWTClaimsSet claimsSet = jwtProcessor.process(jwt, null)
 
+            accessTokenRoles = getRoles(claimsSet)
+
             Scope scope = Scope.parse(claimsSet.getClaim(OidcConfiguration.SCOPE))
```

Comment:
Actually this will return a list of scopes. not just one.
<img width="541" alt="image" src="https://user-images.githubusercontent.com/72474143/208333419-753c9ac2-8ad5-45b0-862f-73f0f96748c8.png">


Cas based token
<img width="513" alt="image" src="https://user-images.githubusercontent.com/72474143/208333488-2be9359d-661a-4c3a-b069-65f1d26d3724.png">

Reply:
Scope.parse has overloads for both Collection<String> and tokenised Strings using either space or comma, so this is fine.